hack/rules-and-dashboards/sync_rules.py

```diff
@@ -281,6 +290,7 @@ def write_group_to_file(group, url, charts):
             content += "{{- $additionalGroupByLabels := append $Values.defaultRules.additionalGroupByLabels $clusterLabel }}\n"
             content += '{{- $groupLabels := join "," $additionalGroupByLabels }}\n'
             content += "{{- $grafanaHost := ternary (index (($Values.grafana).ingress).hosts 0) (($Values.external).grafana).host ($Values.grafana).enabled }}\n"
+            content += '{{- $jobLabel := (index $Values "prometheus-node-exporter").service.labels.jobLabel | default "node-exporter" }}\n'
```

Comment:
this should not be the same for all rules since not all of them are for node-exporter